8kyu/area-or-perimeter.js

```diff
@@ -0,0 +1,4 @@
+const areaOrPerimeter = function(l, w) {
```

Comment:
Consider changing this to an arrow function.
```suggestion
const areaOrPerimeter = (l, w) => (l === w) ? l * w : 2 * (l + w);
```

Reply:
This is done